nixos/modules/services/misc/packagekit.nix

```diff
@@ -34,29 +34,13 @@ in
           installing software. Software utilizing PackageKit can install
           software regardless of the package manager.
         '';
-
-      # TODO: integrate with PolicyKit if the nix backend matures to the point
-      # where it will require elevated permissions
-      backend = mkOption {
-        type = types.enum [ "test_nop" ];
-        default = "test_nop";
-        description = ''
-          PackageKit supports multiple different backends and <literal>auto</literal> which
-          should do the right thing.
-          </para>
-          <para>
-          On NixOS however, we do not have a backend compatible with nix 2.0
-          (refer to <link xlink:href="https://github.com/NixOS/nix/issues/233">this issue</link> so we have to force
-          it to <literal>test_nop</literal> for now.
-        '';
-      };
     };
   };
```

Comment:
I guess it doesn't make sense to have this option anymore. But you'll have to use `mkRemovedOptionModule`